core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -503,24 +537,29 @@ object StorageTool extends Logging {
   }
 
   def formatCommand(
-    stream: PrintStream,
-    directories: Seq[String],
-    metaProperties: MetaProperties,
-    metadataVersion: MetadataVersion,
-    ignoreFormatted: Boolean
-  ): Int = {
+                     stream: PrintStream,
+                     directories: Seq[String],
+                     metaProperties: MetaProperties,
+                     metadataVersion: MetadataVersion,
+                     ignoreFormatted: Boolean,
+                     advertisedListenerEndpoints: scala.collection.Seq[kafka.cluster.EndPoint],
+                     controllersQuorumVoters: String
+                   ): Int = {
```

Comment:
Please preserve the old indentation. In Kafka that is how we indent parameters to a method that span multiple lines.